From 5bca25c62473aa91654c884461585698ef4bb9d9 Mon Sep 17 00:00:00 2001
From: Mike Gerdts <mike.gerdts@joyent.com>
Date: Mon, 6 May 2019 14:21:20 -0500
Subject: [PATCH] TOOLS-2252 prototypes in eng.git should follow RFD 164
 Reviewed by: Patrick Mooney <patrick.mooney@joyent.com> Reviewed by: Tim
 Foster <tim.foster@joyent.com> Approved by: John Levon
 <john.levon@joyent.com>

---
 prototypes/prototype.Makefile | 2 +-
 prototypes/prototype.bash     | 2 +-
 prototypes/prototype.c        | 2 +-
 prototypes/prototype.css      | 2 +-
 prototypes/prototype.d        | 2 +-
 prototypes/prototype.h        | 2 +-
 prototypes/prototype.js       | 2 +-
 prototypes/prototype.ksh      | 2 +-
 prototypes/prototype.md       | 2 +-
 prototypes/prototype.restdown | 2 +-
 prototypes/prototype.rs       | 7 +++++++
 prototypes/prototype.sh       | 2 +-
 12 files changed, 18 insertions(+), 11 deletions(-)
 create mode 100644 prototypes/prototype.rs

diff --git a/prototypes/prototype.Makefile b/prototypes/prototype.Makefile
index 0957826..4225d24 100644
--- a/prototypes/prototype.Makefile
+++ b/prototypes/prototype.Makefile
@@ -5,7 +5,7 @@
 #
 
 #
-# Copyright (c) 2014, Joyent, Inc.
+# Copyright 2019 Joyent, Inc.
 #
 
 #
diff --git a/prototypes/prototype.bash b/prototypes/prototype.bash
index dd305aa..3ff0f1f 100644
--- a/prototypes/prototype.bash
+++ b/prototypes/prototype.bash
@@ -6,7 +6,7 @@
 #
 
 #
-# Copyright (c) 2014, Joyent, Inc.
+# Copyright 2019 Joyent, Inc.
 #
 
 #
diff --git a/prototypes/prototype.c b/prototypes/prototype.c
index 3297ea9..7ce1abf 100644
--- a/prototypes/prototype.c
+++ b/prototypes/prototype.c
@@ -5,7 +5,7 @@
  */
 
 /*
- * Copyright (c) 2014, Joyent, Inc.
+ * Copyright 2019 Joyent, Inc.
  */
 
 /*
diff --git a/prototypes/prototype.css b/prototypes/prototype.css
index 3297ea9..7ce1abf 100644
--- a/prototypes/prototype.css
+++ b/prototypes/prototype.css
@@ -5,7 +5,7 @@
  */
 
 /*
- * Copyright (c) 2014, Joyent, Inc.
+ * Copyright 2019 Joyent, Inc.
  */
 
 /*
diff --git a/prototypes/prototype.d b/prototypes/prototype.d
index 941effb..da30dac 100644
--- a/prototypes/prototype.d
+++ b/prototypes/prototype.d
@@ -7,7 +7,7 @@
  */
 
 /*
- * Copyright (c) 2014, Joyent, Inc.
+ * Copyright 2019 Joyent, Inc.
  */
 
 /*
diff --git a/prototypes/prototype.h b/prototypes/prototype.h
index 624c8be..3644f8f 100644
--- a/prototypes/prototype.h
+++ b/prototypes/prototype.h
@@ -5,7 +5,7 @@
  */
 
 /*
- * Copyright (c) 2014, Joyent, Inc.
+ * Copyright 2019 Joyent, Inc.
  */
 
 #ifndef _PROTOTYPE_H
diff --git a/prototypes/prototype.js b/prototypes/prototype.js
index 3297ea9..7ce1abf 100644
--- a/prototypes/prototype.js
+++ b/prototypes/prototype.js
@@ -5,7 +5,7 @@
  */
 
 /*
- * Copyright (c) 2014, Joyent, Inc.
+ * Copyright 2019 Joyent, Inc.
  */
 
 /*
diff --git a/prototypes/prototype.ksh b/prototypes/prototype.ksh
index a38f226..74c9e6f 100644
--- a/prototypes/prototype.ksh
+++ b/prototypes/prototype.ksh
@@ -6,7 +6,7 @@
 #
 
 #
-# Copyright (c) 2014, Joyent, Inc.
+# Copyright 2019 Joyent, Inc.
 #
 
 #
diff --git a/prototypes/prototype.md b/prototypes/prototype.md
index 7f5f9dc..acea587 100644
--- a/prototypes/prototype.md
+++ b/prototypes/prototype.md
@@ -7,6 +7,6 @@
 -->
 
 <!--
-    Copyright (c) 2014, Joyent, Inc.
+    Copyright 2019 Joyent, Inc.
 -->
 
diff --git a/prototypes/prototype.restdown b/prototypes/prototype.restdown
index e3c352c..3bec307 100644
--- a/prototypes/prototype.restdown
+++ b/prototypes/prototype.restdown
@@ -10,6 +10,6 @@ apisections: Something, Useful
 -->
 
 <!--
-    Copyright (c) 2014, Joyent, Inc.
+    Copyright 2019 Joyent, Inc.
 -->
 
diff --git a/prototypes/prototype.rs b/prototypes/prototype.rs
new file mode 100644
index 0000000..7ca3f8b
--- /dev/null
+++ b/prototypes/prototype.rs
@@ -0,0 +1,7 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+// Copyright 2019 Joyent, Inc.
+
+//! Describe the purpose of this file.
diff --git a/prototypes/prototype.sh b/prototypes/prototype.sh
index 181223a..aab067d 100644
--- a/prototypes/prototype.sh
+++ b/prototypes/prototype.sh
@@ -6,7 +6,7 @@
 #
 
 #
-# Copyright (c) 2014, Joyent, Inc.
+# Copyright 2019 Joyent, Inc.
 #
 
 #
-- 
2.21.0

